### PR TITLE
Create lsblk.md

### PR DESCRIPTION
**Pull request by gautamkrishnar**
lsblk man

### CODE REVIEW

**Comment by rubenvereecken** [comment]
You managed to upset Travis a bit, check out the details to find out where the linter got annoyed. The html complaints are erroneous, you can ignore those.

On a more serious note, while I see some really good examples listed the page is simply way too long. In our [contributing guidelines](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines) we suggest 5-6 examples, though we've gone a bit over that for large commands. This is not such a one I'm afraid. In fact, I see you listed more examples than my MAN page list usages, like the exotic `-S` option.

Our goal with tldr pages is to have common usage patterns quickly accessible, without having to skim through too much. Pages like that would deter users, and while they're really good for extensive examples, they're pretty much what tldr tries to solve. 

There's some great stuff in here, so I suggest you think over what commands would be used most often, or at least by newbies. For example, I've only ever used the `-a` option. 

I'd be more than happy to review a slimmed-down version of this file.

**Comment by gautamkrishnar** [comment]
@rubenvereecken Ok :+1:

**Comment by gautamkrishnar** [comment]
@rubenvereecken is it ok now :smile:

File: pages/linux/lsblk.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# lsblk
+
+> lsblk lists information about all available or the specified storage devices.
```
Comment (rubenvereecken) [inline pages/linux/lsblk.md]: Could we shorten this into "lists information about storage devices"? That sounds snappier while still in essence containing the above.

File: pages/linux/lsblk.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- General syntax
```
Comment (rubenvereecken) [inline pages/linux/lsblk.md]: We don't do general syntax, just examples. Syntax is a bit contained in those.

File: pages/linux/lsblk.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- Output info about filesystems.
+
+`lsblk −f`
+
+- Use ASCII characters for tree formatting.
+
+`lsblk −i`
+
+- Produce output in the form of a list.
```
Comment (rubenvereecken) [inline pages/linux/lsblk.md]: So just out of curiosity: this option gives me exactly the same as without. Is the list form the default one?

**Comment by rubenvereecken** [comment]
@gautamkrishnar thanks for swiftly trimming it down. Let's just go over the inline comments and all should be good :).

**Comment by gautamkrishnar** [comment]
@rubenvereecken done :+1: .

**Comment by rubenvereecken** [comment]
We don't have guidelines for line formatting yet (capitalization/end with punctuation) but you're being inconsistent there with 2/6 having no periods at the end. If you can just patch those up you have my heartfelt :+1:

**Comment by gautamkrishnar** [comment]
@rubenvereecken :+1:

**Comment by gautamkrishnar** [comment]
@rubenvereecken squashed

**Comment by rubenvereecken** [comment]
Great, thanks. We'll fix the syntax with the formatter.

Thanks for the page!

**Pull request merged**